frontend/src/common/ApolloGraphQLClient.ts

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
```

Comment:
I agree with this code but I wonder if it should not be in sirius-web instead.

For example, I don't see why we should have anything to do with credentials support. Especially since different users of `sirius-components` already have different authentication strategies.

Reply:
I understand the concern, but this would lead to duplicate code in all apps based on `sirius-components`. I'll see if we can tweak the `ApolloClient` after it has been constructed, so that we can provide here one with some default config, and apps that need it could add the `credentials` part if they need it.

Note that the current custom hooks already mix in some authentication-related code, e.g.:
* https://github.com/eclipse-sirius/sirius-components/blob/master/frontend/src/common/GraphQLHooks.ts#L185
* https://github.com/eclipse-sirius/sirius-components/blob/master/frontend/src/common/GraphQLHttpClient.ts#L26 (hard-coded `credentials: 'include',  mode: 'cors'`).

Reply:
Fixed by https://github.com/eclipse-sirius/sirius-components/pull/128/commits/4a118e75b505730d3251d0e596a894ce719a727e, and https://github.com/eclipse-sirius/sirius-web/pull/12/commits/8d298629cbd08798d4678fe0bedf158e4d732f6f on the sirius-web side.